docker/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG userid=10001
 
 WORKDIR /app/
 RUN groupadd --gid $groupid app && \
-    useradd -g app --uid $userid --shell /usr/sbin/nologin app
+    useradd -g app --uid $userid --shell /usr/sbin/nologin --create-home app
```

Comment:
`npm` gets mighty fussy if `$HOME` doesn't exist. I figured we might as well create it. That saves us from dealing with getting around whatever it is `npm` is doing.